### ISSUE TITLE
Re-scan after edits

### ISSUE THREAD

**Issue by thiagoarrais** [jimschubert/auto-labeler#3]
Sometimes I forget to add a tag to an issue title when creating it. Can the bot re-scan the issue after editing just like it does after creation?

I can always re-create the issue (instead of editing). But that doesn't seem very ergonomic.

I'm open to offering a pull request. Just let me know if you're interested in this feature.

**Comment by jimschubert** [comment]
Yeah, I'd be interested in this feature if you're able to put it into a PR. I'm currently working on other things, and switching jobs as well, so it would be some time before I could get around to it.

I would think responding to edit could get wordy if the bot posts when no labels are added (which it currently does). That may not be a huge issue if TODO checkboxes don't count as edits, and the event only fires on explicit user edits.
